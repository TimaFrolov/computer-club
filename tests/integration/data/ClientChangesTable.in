2
01:00 02:00
7
01:15 1 client1
01:20 2 client1 1
01:25 2 client1 2
