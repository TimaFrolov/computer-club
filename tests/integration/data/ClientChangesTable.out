01:00
01:15 1 client1
01:20 2 client1 1
01:25 2 client1 2
02:00 11 client1
02:00
1 7 00:05
2 7 00:35
